src/main/groovy/org/jetbrains/intellij/PrepareSandboxTask.groovy

```diff
@@ -24,12 +26,15 @@ class PrepareSandboxTask extends Sync {
         group = IntelliJPlugin.GROUP_NAME
         description = "Creates a folder containing the plugins to run Intellij IDEA with."
 
+        def extension = project.extensions.findByName(IntelliJPlugin.EXTENSION_NAME) as IntelliJPluginExtension
+
         CopySpecInternal plugin = rootSpec.addChild()
-        classes = plugin.addChild().into("classes")
-        libraries = plugin.addChild().into("lib")
-        metaInf = plugin.addChild().into("META-INF")
+        classes = plugin.addChild().into("$extension.pluginName/classes")
+        libraries = plugin.addChild().into("$extension.pluginName/lib")
+        metaInf = plugin.addChild().into("$extension.pluginName/META-INF")
+
+        externalPlugins = rootSpec.into(Utils.pluginsDir(extension, inTests)).from("externalPluginsLibs").exclude("*.zip")
```

Comment:
At the moment all dependencies are packet into distribution zip. I'll fix that in next commits.